src/openfermion/resource_estimates/pbc/thc/utils/gvec_logic.py

```diff
@@ -0,0 +1,184 @@
+# coverage: ignore
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import itertools
+import numpy as np
+from pyscf.lib.numpy_helper import cartesian_prod
+
+
+def get_miller_indices(kmesh):
+    """Calculate the miller indices on a gamma centered non-1stBZ Monkorhst-Pack
+      mesh
```

Comment:
here and elsewhere in the file: really try to keep the first line of the docstring to be one line, followed by a blank line, and then the rest of the docstring. This affects docstring rendering in our and other doc tools

Reply:
Sorry! There was some mishmash of docstring conventions going on. Lesson learned.